Controllers/AuthenticationController.py

```diff
@@ -1,21 +1,29 @@
 from flask import Blueprint, request
 from Response import Response
 from Services.AuthenticationService import AuthenticationService
+from Models.User import User
+from Models.Session import Session
+from flask_mail import Message
+from flask import current_app as app
+from flask_mail import Mail
```

Comment:
Make Message and Mail one import

Reply:
makes sense we dont wanna fun flask_mail twice